### DISCUSSION TITLE
Usability: Font size of the menu text

### DISCUSSION THREAD

**Discussion by Hermann12** [geany/geany#3744] [General]
Is there a possibility to increase the font size of the gui menu size for people with bad eyes.
If there is any configuration possible, please let me know.

**Comment by elextr** [comment]
The style of everything except the editing pane is set by the GTK theme, depending on your OS it might be as simple as selecting a large text setting under the accessibility heading (although that will do the whole UI).

Otherwise you can edit the GTK CSS that controls the theme, or limit it to Geany by `Tools->Configuration Files->geany.css` but of course you need to learn GTK CSS (a subset of "proper" internet CSS).

**Reply by Hermann12** [reply]
Thanks, I found this post now; https://github.com/geany/geany/discussions/3653#discussioncomment-7374788 and insert * { -gtk-dpi: 150; } into geany.css solves my problem.
